spec/fizzbuzz_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'fizzbuzz'
+
+describe Fizzbuzz do
+  let(:fizzbuzz) { Fizzbuzz.new(3, 5, 15) }
+  let(:number_string) { '1 2 3 4 5 6 7 8 9 10 11 12 13 14 15' }
```

Comment:
when you have number_array then number_string can look like

```ruby
   let(:number_string) { number_arrray.join(' ') }
```

similar with result_string